src/SFA.DAS.ApplyService.Web/Views/Application/PagesWithSections.cshtml

```diff
@@ -25,25 +25,23 @@
             else
             {
                 <h1 class="govuk-heading-xl">@Model.Title</h1>
-                <ul class="govuk-list">
+
+                <ol class="app-task-list">
                     <li>
-                        <ul class="app-task-list govuk-!-margin-bottom-7">
+                        <h2 class="app-task-list__section">@Html.Raw(pagesToCheck[0].Title)</h2>
+                        <ul class="app-task-list__items app-task-list__items--no-number">
                             @for (var i = 0; i < pagesToCheck.Count; i++)
                             {
                                 var pg = pagesToCheck[i];
                                 isSamePage = pageTitle == pg.Title;
                                 pageTitle = pg.Title;
                                 nextPageTitle = "";
+
                                 if (i < pagesToCheck.Count - 1)
                                 {
                                     nextPageTitle = (pagesToCheck[i + 1]).Title;
                                 }
 
-                                @if (!isSamePage)
```

Comment:
are you sure this should go?

Reply:
I've removed all reference to that now. I can't see any issue with the UI on first glance but lets see what comes out of testing.